kong/plugins/oauth2/daos.lua

```diff
@@ -8,6 +9,21 @@ local function generate_if_missing(v, t, column)
   return true
 end
 
+local function validate_uris(v, t, column)
+  if v and type(v) == "table" then
```

Comment:
@PGBI correct me if I am wrong, but since `redirect_uri` is now an `array` in the schema, we don't have to check if it's a table. It should always be parsed like a table, or `nil`.

So `if v and type(v) == "table" then` becomes `if v then` and that should be enough.